src/taskgraph/config.py

```diff
@@ -35,6 +35,7 @@
                 "lowest",
             ),
         ),
+        Optional("task-deadline-after"): optionally_keyed_by("project", str),
```

Comment:
Might be nice to add the `description` field here to describe the format it expects :)

Reply:
That's a good idea, yeah.